src/pages/General.vue

```diff
@@ -1,12 +1,20 @@
 <router lang="yaml">
-  name: Welcome
+  name: General
 </router>
 <template>
   <div>
-    <div class="welcome">
-      <p>
-        Welcome...
+    <div class="general">
+      <h1>
+        Welcome to Rancher Desktop
+      </h1>
+      <p>Rancher Desktop provides Kubernetes and image management through the use of a desktop application.</p>
+      <ul>
+        <li>Project Status: <i>alpha</i></li>
+        <li>Project Discussions: #rancher-desktop in <a href="#" onclick="require('electron').shell.openExternal('https://slack.rancher.io/')">Rancher Users</a> Slack</li>
```

Comment:
We should probably get a helper function for link clicking? Or even better, set up a [`will-navigate`](https://www.electronjs.org/docs/api/web-contents#event-will-navigate) event handler to open all external links externally?

The only links we'd open internally are the navigation things, which shouldn't trigger `will-navigate`, just `did-navigate-in-page`, right?

Reply:
Filed #231 to handle this afterwards